pulsar-client-reactive-jackson/src/test/java/org/apache/pulsar/reactive/client/jackson/PulsarReactiveClientModuleTests.java

```diff
@@ -493,6 +508,13 @@ void shouldSerDeserEmptyDeadLetterPolicy() throws Exception {
 		assertThat(json).isEqualTo("{\"maxRedeliverCount\":0}");
 	}
 
+	@Test
+	void shouldSerDeserTestRedeliveryBackoff() throws Exception {
```

Comment:
I believe this should be named `shouldSerDeserEmptyRedeliveryBackoff`.

Reply:
Right, fixed.